content/docs/04-clusters/02-data-center/01-maas/15-create-manage-maas-clusters.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "Create and Manage MAAS Clusters"
+metaTitle: "Create and Manage MAAS Clusters"
+metaDescription: "Learn how to create and manage MAAS clusters in Palette."
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+
+# Overview 
+
+Palette supports creating and managing Kubernetes clusters deployed to a MAAS account. This section guides you on how to create a Kubernetes cluster in MAAS that is managed by Palette.
+
+# Prerequisites
+
+- An installed Private Cloud Gateway (PCG). Review [Install and Manage MAAS Gateway](/clusters/data-center/maas/install-manage-maas-pcg) for guidance.
+
+
+- At least one preconfigured MAAS cloud account with credentials.
+
+
+- A cluster profile for the MAAS environment. Review [Cluster Profiles](/cluster-profiles) for more information. 
+
+# Deploy a MAAS cluster
+
+To deploy a new MAAS cluster:
+
+<br />
+
+1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
+
+
+2. Navigate to the **Main Menu** and click **Clusters**. Then click the **Add New Cluster** button.
+
+
+3. Click **Deploy New Cluster** on the Create a New Cluster page.
+
+
+4. Select **MAAS** and click the **Start MAAS Configuration** button.
+
+
+5. Provide basic cluster information: **Cluster name**, **Description**, and **Tags**.
+
+
+6. Select your cloud account from the **drop-down Menu** and click **Next**.
+
+
+7. Select the cluster profile for your MAAS cluster. 
```

Comment:
you will of course have needed to create a maas cluster profile first, and this should be a prerequisite (?)